tools/clang_tidy/test/clang_tidy_test.dart

```diff
@@ -106,7 +106,7 @@ Future<int> main(List<String> args) async {
       <String>[
         '--compile-commands',
         // This just has to exist.
-        io.Platform.executable,
+        io.Platform.script.path,
```

Comment:
`io.Platform.executable` crashed, this file is actually parsed now and needs to be utf8.

Reply:
Could you update the comment on the line above?

Reply:
Done.